FeedStoreChallenge/CoreData/ManagedCache.swift

```diff
@@ -0,0 +1,24 @@
+import CoreData
+
+@objc(ManagedCache)
+internal class ManagedCache: NSManagedObject {
```

Comment:
This class is not intended to be inherited thus it should be `final`
Please make the class `final`.

```suggestion
final class ManagedCache: NSManagedObject {
```